cli/command/stack/deploy_bundlefile.go

```diff
@@ -34,6 +36,11 @@ func deployBundle(ctx context.Context, dockerCli *command.DockerCli, opts deploy
 	for internalName, service := range bundle.Services {
 		name := namespace.Scope(internalName)
 
+		em := "vip"
+		if service.EndpointMode != nil {
+			em = *service.EndpointMode
+		}
```

Comment:
I don't think we should assume a default on the client. Different backends may want different defaults. Can we leave the default as `""` ?